card/mod/01_history/set/all/content_history.rb

```diff
@@ -100,7 +102,7 @@ def last_actor
 end
 
 def last_act
-  if action = last_action
+  if (action = last_action)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.